components/ILIAS/Init/classes/class.ilStartUpGUI.php

```diff
@@ -1379,7 +1379,7 @@ private function doLogout(): void
         ilUtil::setCookie("ilClientId", "");
 
         $this->ctrl->setParameter($this, 'lang', $user_language);
-        $this->ctrl->setParameter($this, 'client_id', $client_id);
+        $this->ctrl->setParameter($this, 'client_id', CLIENT_ID);
```

Comment:
Do we need this change?

Reply:
Yes, we need this change, because $client_id is undefined.